models/prices/prices_tokens.sql

```diff
@@ -1619,6 +1619,7 @@ VALUES
     ("xen-xen-crypto", "ethereum", "XEN", "0x06450dee7fd2fb8e39061434babcfc05599a6fb8", 18),
     ("bob-bob", "ethereum", "BOB", "0xb0b195aefa3650a6908f15cdac7d92f8a5791b0b", 18),
     ("unbnk-unbanked", "ethereum", "UNBNK", "0x06b884e60794ce02aafab13791b59a2e6a07442f", 18),
+    ("fin-definer", "ethereum", "FIN", "0x054f76beed60ab6dbeb23502178c52d6c5debe40", 18),
```

Comment:
It seems `fin-definer` not existed on coinpaprika api:
https://api.coinpaprika.com/v1/coins/fin-definer

Reply:
Please try again with capital “D” on coinpaprika

Reply:
https://coinpaprika.com/coin/definer-fin/

Reply:
Do you have a specific link?

Reply:
Could you check this one?

Reply:
https://coinpaprika.com/coin/definer-fin/